src/base/static/components/input-form/publish-control-toolbar.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import cx from "bem-classnames";
+
+import { RadioBigButton } from "../input-form/radio-big-button";
```

Comment:
If we refactor the export, this be:

`import RadioBigButton from "../input-form/radio-big-button";`

the exception to object wrapped export would be utility functions

Reply:
yes, I'll clean this up throughout the PR